Formula/z/zsh-fast-syntax-highlighting.rb

```diff
@@ -19,7 +19,7 @@ class ZshFastSyntaxHighlighting < Formula
   uses_from_macos "zsh" => [:build, :test]
 
   def install
-    pkgshare.install Dir["*"]
+    pkgshare.install Dir["*", ".fast-*"]
```

Comment:
```suggestion
    pkgshare.install buildpath.glob("*", File::FNM_DOTMATCH)
```
Let's just match all directories that start with a `.`.

Reply:
Tried the above locally, but it gives an error:
```zsh
HOMEBREW_NO_INSTALL_FROM_API=1 brew reinstall --build-from-source zsh-fast-syntax-highlighting
# [...]
# Error: An exception occurred within a child process:
#   Errno::ENOENT: No such file or directory - /private/tmp/zsh-fast-syntax-highlighting-20250724-92047-8hdy9m/fast-syntax-highlighting-1.56/.brew_home
```

Reply:
Thanks for testing! Try:

```suggestion
    to_install = buildpath.glob("*", File::FNM_DOTMATCH).reject do |pn|
      pn.basename == Pathname.new(".brew_home")
    end
    pkgshare.install to_install
```